statsmodels/discrete/tests/test_constrained.py

```diff
@@ -449,6 +449,31 @@ def setup_class(cls):
         cls.res1 = fit_constrained(mod1, R, q)
 
 
+class TestLogitConstrained1(CheckGLMConstrainedMixin):
+
+    @classmethod
+    def setup_class(cls):
+        cls.idx = slice(None)
```

Comment:
if this isn't overriden elsewhere, can you set it at the class level outside of setup_class?  These are easier to reason about when they are less stateful

Reply:
defensive programming, 
There is no requirement that it is not changed somewhere, and I don't have to think about inheritance for class attributes when adding subclasses.